apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts

```diff
@@ -192,6 +192,9 @@ const blockLinks: { [ key: string ]: string } = {
 	'jetpack/mailchimp': 'https://wordpress.com/support/wordpress-editor/blocks/mailchimp-block/',
 
 	'jetpack/paywall': 'https://wordpress.com/support/paid-newsletters/#use-the-paywall-block',
+
+	'jetpack/sharing-buttons':
+		'https://wordpress.com/support/wordpress-editor/blocks/sharing-buttons-block/',
```

Comment:
The link doesn't seem to work. Do we need to create this support article first?

Reply:
Thanks for the review. The documentation in general is still not ready, but we would make sure that the feature will go live after we have one (task https://github.com/Automattic/vulcan/issues/142).
This is a part of Call For Testing pdtkmj-2dQ-p2.